scriptTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ "$1" = "install" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
+	sudo python3.4 setup.py build
+	echo "=========== Installed and built the sniffles ========="
+	echo
+elif [ "$1" = "test" ]; then
+	clear
+	echo "=========== Begin to install and build ========="
+	sudo python3.4 setup.py install
```

Comment:
everytime i implemented a new feature, i need to build it and reinstall sniffles to test if the program crashs, the scriptTest is just more convinient, maybe i will not push it into origin